README.md

```diff
@@ -23,10 +23,22 @@ The following components are available in this repository:
 - [Loader Documentation](docs/LoaderInterfaceArchitecture.md)
 - [Tests](tests/)
 
+### Vulkan SC
+
+Vulkan SC is an API derived from Vulkan with various changes applied to fulfill the special requirements of safety critical applications.
+The Vulkan SC API has a large overlap with Vulkan.
+As such, we endeavor to leverage as many of the Vulkan Ecosystem components as possible.
+
+This repository contains the Vulkan SC Loader which is intended for **development** environments,
+and is not expected to be used in **production** (safety) environments.
+
+The `sc_main` branch is based on the `sdk-1.2.198` Vulkan loader tag and contains Vulkan SC modifications to support the Vulkan SC 1.0 API.
+
 ## Contact Information
 
 - [Charles Giessen](mailto:charles@lunarg.com)
```

Comment:
Is it your intention to leave these people as contacts for the loader? Wouldn't it be better to have Daniel Rakos?

Reply:
> Is it your intention to leave these people as contacts for the loader

Yes. I added myself as contact for Vulkan SC specific issues.
We don't have any other agreed plan for other maintainers.

Reply:
done.